app/src/logger/index.ts

```diff
@@ -26,4 +26,8 @@ export function getLogger(label: string): Logger {
     return defaultLogger.child({label});
 }
 
+export function getLogsEnabled() {
```

Comment:
@microwavekonijn Any suggestions? Brain farting how to typehint this

Reply:
I wouldn't. Also why? I didn't see an issue else I could have saved you a lot of time. https://github.com/winstonjs/winston#filtering-info-objects

Reply:
```js
export function filter(list: string[], whitelist = false): (info: ???) => boolean {
  const set = new Set<string>(list);

  if (whitelist) return (info: ???) => set.has(info.label) ? info : false;
  return (info: ???) => set.has(info.label) ? false : info;
}
```

Reply:
Basically the point of this PR is to control logging flags, so when it comes to looking at logs on life it doesn't print to screen constantly. We probably rather than doing this could implement logging drivers, write most things to text but only certain things to screen. Will need a logging driver for Discord, this is kinda a temporary measure.

Reply:
I understand, but it is not the responsibility of the module to determine if something should be logged or not. With Winston you can have multiple log outputs(transports, e.g. console, discord, a file, etc), each of them share a global configuration and can have their own. Filtering should be done by the logger which is what that function is for. You pass it in the config of the logger when you create it.

Reply:
Also if you flatten the logsEnabled object you can transform it into an array like:
```js 
const list = Object.keys(config.logger.logsEnabled).keys().filter(k => config.logger.logsEnabled[k]);
```